pkg/agent/openflow/client.go

```diff
@@ -738,6 +738,7 @@ func (c *client) InstallTraceflowFlows(dataplaneTag uint8) error {
 	flows := []binding.Flow{}
 	c.conjMatchFlowLock.Lock()
 	defer c.conjMatchFlowLock.Unlock()
+	// Copy default drop rules
 	for _, ctx := range c.globalConjMatchFlowCache {
 		if ctx.dropFlow != nil {
 			flows = append(
```

Comment:
Having discussion with @weiqiangt , after metric flow is introduced, the actual "drop" action should happen in the XgressMetricTable, so I think we could add traceflow only in XgressMetricTable with the flow having "dropFlag". Otherwise, the flow in the default drop table is useless.

Reply:
Think over, as metricflow doesn't record from which policy table the packet is resubmitted in, does it affect the result of Traceflow? I mean we actually want to report that the packet is dropped by K8s Policy Rule or Antrea Policy Rule, but it seems the current implementation would report the packet is dropped by MetricTable

Reply:
Will the K8s NetworkPolicy use default drop table?
I think Traceflow need to mock all NetworkPolicy flows with "drop" action, including default drop flow for K8s NetworkPolicy and XgressMetricTable for Antrea NetworkPolicy.

Reply:
We can identify this by the prefix in networkPolicy property. This function is added by @tnqn 
Here is my test result:
```
  status:
    phase: Succeeded
    results:
    - node: gran-k8s0-2
      observations:
      - action: Forwarded
        component: SpoofGuard
      - action: Forwarded
        component: NetworkPolicy
        componentInfo: EgressRule
        networkPolicy: AntreaNetworkPolicy:default/test-anp
      - action: Dropped
        component: NetworkPolicy
        componentInfo: IngressMetric
        networkPolicy: AntreaNetworkPolicy:default/test-anp
      timestamp: 1603965404
```